java/LICENSE-bundled

```diff
@@ -1,26 +0,0 @@
-The binary distribution of this product bundles binaries of Boost
-which are available under the following license:
-
-Boost Software License - Version 1.0 - August 17th, 2003
```

Comment:
@kkraus14 is it ok to remove this LICENSE?

Reply:
I would defer to @rapidsai/cudf-java-codeowners

Reply:
@sameerz @JasonAtNvidia ?

Reply:
Yes, this LICENSE should be removed if we are no longer linking against boost.  The Java bindings currently are building libcudf requesting a statically linked boost filesystem library to avoid having problems locating the correct version of the library at runtime.